packages/shared/styles/components/organisms/SfMegaMenu.scss

```diff
@@ -1,120 +1,115 @@
 @import "../../helpers";
 
 %menu-item {
-  padding: var(--mega-menu-item-padding, var(--spacer-big));
-  border-bottom: var(--mega-menu-item-border-bottom, 1px solid var(--c-light));
-  font: var(
-    --mega-menu-item-font,
-    font(
-      --mega-menu-item,
-      300,
-      var(--font-size-big),
-      1.6,
-      var(--font-family-secondary)
-    )
-  );
-  text-transform: var(--mega-menu-item-text-transform);
+  --menu-item-label-color: var(--c-black);
+  --menu-item-padding: var(--spacer-sm);
+  --menu-item-border-width: 0 0 1px 0;
+  --menu-item-font-weight: var(--font-light);
+  --menu-item-letter-spacing: 0.1px;
+  --menu-item-title-color: var(--c-black);
+
   @include for-desktop {
-    --mega-menu-item-font-size: var(--font-size-regular);
-    --mega-menu-item-padding: 0 0 10px 0;
-    --mega-menu-item-border-bottom: 0;
+    --menu-item-title-font-weight: var(--font-bold);
+    --menu-item-label-color: var(--c-dark-variant);
+    --menu-item-padding: 0 0 var(--spacer-sm) 0;
+    --menu-item-border-width: 0;
   }
 }
+
 .sf-mega-menu-column {
-  margin: var(--mega-menu-column-margin);
-  &__title {
-    @extend %menu-item;
-    --mega-menu-item-font-weight: 500;
-    --mega-menu-item-padding: var(--spacer-big);
-    margin: var(--mega-menu-column-title-margin);
-  }
-  & > .sf-menu-item {
-    @extend %menu-item;
-    &--primary {
-      --mega-menu-item-font-weight: 500;
-    }
-  }
+  @extend %menu-item;
+
   @include for-desktop {
-    --mega-menu-column-margin: 0 var(--spacer-extra-big) 0 0;
-    &__title {
-      --mega-menu-item-padding: 0 0 20px 0;
-      --mega-menu-item-text-transform: uppercase;
+    margin: var(--mega-menu-column-margin, 0 var(--spacer-xl) 0 0);
+    & :first-child {
+      --menu-item-padding: 0 0 var(--spacer-base) 0;
+      --menu-item-text-transform: uppercase;
+    }
+    &:active,
+    &:hover {
```

Comment:
I think` :hover` should be applied to each list item separately instead of a column

Reply:
According to the design each hovered column should be highlighted, but I agree with you. What should we do?  @aniamusial @pspaczek